src/Tribe/PUE/Notices.php

```diff
@@ -351,19 +351,20 @@ protected function find_your_key_text() {
 	 * @return string
 	 */
 	protected function get_formatted_plugin_names( $group ) {
-		$num_plugins = count( $this->notices[ $group ] );
-
-		if ( ! $num_plugins ) {
+		if ( ! count( $this->notices[ $group ] ) ) {
 			return '';
 		}
 
 		$plugin_list = array_intersect( $this->registered, array_keys( $this->notices[ $group ] ) );
+		$num_plugins = count( $plugin_list );
 
-		if ( 1 === $num_plugins ) {
+		if ( 0 === $num_plugins ) {
+			return '';
+		}
```

Comment:
Hah, yes we can. I did indeed deliberately break away from our established style here because I find this far more readable, but will adjust.